### PR TITLE
DP3586: Access UCL data refresh

### PR DESCRIPTION
**Pull request by benjiebeetle**

### CODE REVIEW

File: js/app/access-ucl-eligibility-widget.js [hunk 1]
```diff
@@ -262,6 +262,7 @@
                                 var indeterminate = false;
                                 const acorns = ["L","M","O","P","Q"];
                                 if (data) {
+                                    console.log(data);
```
Comment (Sem-the-dev) [inline js/app/access-ucl-eligibility-widget.js:265]: console.log

File: js/appsrc/access-ucl-eligibility-widget.js [hunk 2]
```diff
@@ -262,6 +262,7 @@
                                 var indeterminate = false;
                                 const acorns = ["L","M","O","P","Q"];
                                 if (data) {
+                                    console.log(data);
```
Comment (Sem-the-dev) [inline js/appsrc/access-ucl-eligibility-widget.js:265]: another console.log

File: js/app/access-ucl-eligibility-widget.js [hunk 3]
```diff
@@ -186,7 +186,7 @@
                 });
 
                 var eligibilityData = {};
-                var get_url = (location.href.indexOf("micro.ucl.ac.uk") > -1?"http://micro.ucl.ac.uk/access-ucl/www/eligibility-2022.php":(location.href.indexOf("wwwapps-uat.ucl.ac.uk") > -1?"https://wwwapps-uat.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2022.php":"https://www.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2022.php"));
+                var get_url = (location.href.indexOf("local-micro.lndo.site") > -1?"https://local-micro.lndo.site/access-ucl/www/eligibility-2023.php":(location.href.indexOf("wwwapps-uat.ucl.ac.uk") > -1?"https://wwwapps-uat.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2023.php":"https://www.ucl.ac.uk/digital-presence-services/access-ucl/www/eligibility-2023.php"));
```
Comment (Sem-the-dev) [inline js/app/access-ucl-eligibility-widget.js:189]: How come it's now using the `local-micro.lndo.site` here, the other file in GitLab removed it if I remember correctly

File: js/app/access-ucl-eligibility-widget.js (hunk 1 above)
Comment (benjiebeetle) [reply js/app/access-ucl-eligibility-widget.js]: Well spotted. I've removed it now.

**Review by Sem-the-dev — APPROVED** [review]

File: js/app/access-ucl-eligibility-widget.js (hunk 3 above)
Comment (benjiebeetle) [reply js/app/access-ucl-eligibility-widget.js:189]: This is for local development. This logic allows it to work on local, UAT and production. I used to use a LAMP setup on the domain micro.ucl.ac.uk - but I've replaced this with Lando now, so anyone can use it.

**Pull request merged**